advance-exercises/src/components/01-CompoundComponents.js

```diff
@@ -20,28 +20,48 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 class RadioGroup extends React.Component {
+  constructor(props) {
+    super(props);
+    const { defaultValue } = this.props;
```

Comment:
Should a new empty line be added here, as it separates two logical parts? One was added in the same function.